### PR TITLE
58083: Updated the TLS vs SSL.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/directoryservice.rst [hunk 1]
```diff
@@ -140,9 +140,8 @@ these settings by checking
...
    +--------------------------+---------------+-------------+---------------------------------------------------------------------------------------------------------------------+
-   | Encryption Mode          | drop-down     | ✓           | Choices are *Off*, *SSL*, or *TLS*. Refer to `SSL vs. TLS                                                           |
-   |                          | menu          |             | <https://www.globalsign.com/en/blog/ssl-vs-tls-difference/>`__ for more information on SSL and TLS.                 |
-   |                          |               |             |                                                                                                                     |
+   | Encryption Mode          | drop-down     | ✓           | Choices are *Off*, *SSL*, or *TLS*.                                                                                 |
+   |                          |               |             | Choosing SSL uses LDAPS protocol (port 636). Choosing TLS uses LDAP protocol (port 389).                            |
```
Comment (wblock) [inline userguide/directoryservice.rst]: We don't need to say "choosing" here, it's better with current tense. And single stars around field contents:
```suggestion
   |                          |               |             | *SSL* selects LDAPS protocol (port 636). *TLS* selects LDAP protocol (port 389).                            |
```

File: userguide/directoryservice.rst [hunk 2]
```diff
@@ -573,8 +572,8 @@ Those new to LDAP terminology should read the
    +-------------------------+--------------+-------------+------------------------------------------------------------------------------------------------+
    | Encryption Mode         | drop-down    | ✓           | Choices are *Off*, *SSL*, or *TLS*. Note that either *SSL* or *TLS* and a                      |
    |                         | menu         |             | :guilabel:`Certificate` must be selected for authentication to work.                           |
-   |                         |              |             | Refer to `SSL vs. TLS <https://www.globalsign.com/en/blog/ssl-vs-tls-difference/>`__           |
-   |                         |              |             | for more information on SSL and TLS.                                                           |
+   |                         |              |             | Choosing SSL uses LDAPS protocol (port 636). Choosing TLS uses LDAP protocol (port 389).       |
```
Comment (wblock) [inline userguide/directoryservice.rst]: As above.

Comment (a-stjohn) [inline userguide/directoryservice.rst]:
```suggestion
   |                         |              |             | *SSL* selects LDAPS protocol (port 636). *TLS* selects LDAP protocol (port 389).                 |
```

File: userguide/directoryservice.rst [hunk 3]
```diff
@@ -573,8 +572,8 @@ Those new to LDAP terminology should read the
    +-------------------------+--------------+-------------+------------------------------------------------------------------------------------------------+
    | Encryption Mode         | drop-down    | ✓           | Choices are *Off*, *SSL*, or *TLS*. Note that either *SSL* or *TLS* and a                      |
    |                         | menu         |             | :guilabel:`Certificate` must be selected for authentication to work.                           |
-   |                         |              |             | Refer to `SSL vs. TLS <https://www.globalsign.com/en/blog/ssl-vs-tls-difference/>`__           |
-   |                         |              |             | for more information on SSL and TLS.                                                           |
+   |                         |              |             | *SSL* selects LDAPS protocol (port 636). *TLS* selects LDAP protocol (port 389).                 |
```
Comment (a-stjohn) [inline userguide/directoryservice.rst]:
```suggestion
   |                         |              |             | *SSL* selects LDAPS protocol (port 636). *TLS* selects LDAP protocol (port 389).                |
```

**Pull request merged**